src/elements/element.rectangle.js

```diff
@@ -50,42 +50,76 @@ module.exports = function(Chart) {
 	}
 
 	Chart.elements.Rectangle = Chart.Element.extend({
-		draw: function() {
+		draw: function(isHorizontalBar) {
```

Comment:
I agree. We can do option 1 now and then refactor later. I looked at doing option 2 a while ago and it was a lot of work